### PR TITLE
Add web support for the "device.webtoken.create" event

### PR DESCRIPTION
**Pull request by codingllama**
Map "device.webtoken.create" and "TV008I" in the Web UI.

https://github.com/gravitational/teleport.e/issues/3236

### CODE REVIEW

**Comment by codingllama** [comment]
image

**Review by kimlisa — APPROVED** [review]

**Comment by codingllama** [comment]
Thanks for the quick review, Lisa!

File: web/packages/teleport/src/services/audit/types.ts [hunk 1]
```diff
@@ -120,6 +120,7 @@ export const eventCodes = {
   DEVICE_ENROLL: 'TV005I',
   DEVICE_AUTHENTICATE: 'TV006I',
   DEVICE_UPDATE: 'TV007I',
+  DEVICE_WEB_TOKEN_CREATE: 'TV008I',
```
Comment (zmb3) [inline web/packages/teleport/src/services/audit/types.ts:123]: We usually use a separate code for the failure case. Any reason not to do that here?

Comment (codingllama) [reply web/packages/teleport/src/services/audit/types.ts:123]: All device events are like this. For this particular one we only record successes, so it won't make much of a difference.

**Review by gzdunek — APPROVED** [review]

**Pull request merged**